content/08-cardano-sidechains/03-example-evm-sidechain/05-transferring-tokens.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: Transferring tokens
+metaTitle: Transferring tokens
+---
+
+This tutorial will show you how to transfer `SC_Tokens` between the mainchain and a sidechain.
+
+## Prerequisites
+
+1. We will use the **sidechain-cli** and the **sc-evm-cli** components within this tutorial. Download the binaries and add them to your shell’s path: https://github.com/input-output-hk/sidechains-tooling.
+2. Download the **bech32** conversion tool binary and add to your shell's path: https://github.com/input-output-hk/bech32.
+3. If you have not done so already, create and fund your main chain and sidechain accounts: [Create and funds your accounts](/cardano-sidechains/example-evm-sidechains/create-and-fund-accounts).
```

Comment:
3. If you have not done so already, create and fund your main chain and sidechain accounts: [Create and fund your accounts](/cardano-sidechains/example-evm-sidechains/create-and-fund-accounts)
[funds -> fund]